esp-hal/src/rmt.rs

```diff
@@ -1602,6 +1620,7 @@ pub trait TxChannelInternal: ChannelInternal {
 
     fn is_tx_loopcount_interrupt_set(&self) -> bool;
 
+    #[cfg_attr(place_rmt_driver_in_ram, ram)]
```

Comment:
This is also expected to be inlined, might be worth forcing `#[inline]` instead.

Reply:
Done.